storageClient/files.go

```diff
@@ -0,0 +1,49 @@
+package storageClient
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+)
+
+func (c *StorageClient) UploadFile(url string, encryptedFileName string, checksum string, size int) (int, error) {
```

Comment:
comment, and please extract these parameters into a struct

Reply:
please make these functions in a package called file, not methods for StorageClient.